test/tests/api-cit/v2_0/obm_tests.py

```diff
@@ -57,5 +57,5 @@ def test_setup_snmp_obm_api2(self):
     @depends(after='test_setup_snmp_obm_api2')
     def test_check_snmp_obm_settings(self):
         # """ Checking SNMP OBM settings GET:/nodes """
-        self.assertNotEqual(len(obmSettings().check_nodes(service_type='snmp-obm-service')), 0,
-                            msg='There are missing SNMP OBM settings!')
+        self.assertEqual(len(obmSettings().check_nodes(service_type='snmp-obm-service')), 0,
```

Comment:
The assert and message are proper, even though it looks backwards.  This 'check_nodes' will return a list of nodes that have errors (ie. missing SNMP settings).  if the list is empty (length = 0), that means no errors. 
If the list is not empty, the error message is proper that the listed nodes do not have expected SNMP settings.